day29/goodStringsCount.py

```diff
@@ -0,0 +1,18 @@
+from math import factorial
+
+def nChooseR(n, r):
+    return factorial(n) / (factorial(n-r) * factorial(r))
+
+def goodStringsCount(n):
+    return nChooseR(26, n) * (2**n - n - 1)
```

Comment:
this solution works but it doesn't follow from the examples in your ideas section